Adalpersonur_Lidur2/viewFix.sql

```diff
@@ -0,0 +1,122 @@
+CREATE OR REPLACE VIEW martell.v_pov_characters_human_readable AS
+WITH pov_characters AS (
+    SELECT DISTINCT cb.character_id
+    FROM got.character_books cb
+    WHERE cb.pov = TRUE
+),
+character_details AS (
+    SELECT
+        c.id AS character_id,
+        COALESCE(NULLIF(split_part(array_to_string(c.titles, ','), ',', 1), ''), c.name) || ' ' || c.name AS full_name,
```

Comment:
Ath. þið fáið í þessu tilfelli endurteknu á nöfnum:

```
Chett Chett
Melisandre Melisandre
Merrett Frey Merrett Frey
Brienne of Tarth Brienne of Tarth
Will Will
Samwell Tarly Samwell Tarly
Varamyr Varamyr
Queen of the Andals and the Rhoynar and the First Men Daenerys Targaryen
```

```suggestion
        COALESCE(NULLIF(split_part(array_to_string(c.titles, ','), ',', 1), ''), c.name) || ' ' || c.name AS full_name,
```